modules/appeals_api/config/schemas/v2/200995.json

```diff
@@ -123,25 +123,15 @@
                     }
                   },
                   "required": ["evidenceType"],
-                  "anyOf": [
-                    {
-                      "not": {
-                        "properties": {
-                          "evidenceType": { "contains": { "enum": ["retrieval"] } }
-                        }
-                      }
-                    },
-                    { "required": ["retrieveFrom"] }
-                  ]
-                },
-                "noticeAcknowledgement": {
-                  "enum": [true, false]
+                  "if": { "properties": { "evidenceType": { "const": "retrieval" } } },
+                  "then": { "required": ["retrieveFrom"] }
                 },
+                "noticeAcknowledgement": { "type": "string", "enum": ["yes", "no"] },
```

Comment:
I don't think this should have been an enum to begin with - probably just an oversight. Should just be a `type: boolean` instead.

`"noticeAcknowledgement":  { "type": "boolean" }`

Reply:
Resolved!